frontend/spec/features/digital_objects_spec.rb

```diff
@@ -0,0 +1,368 @@
+    click_on 'Digital Object'
+    element = find('h2')
+    expect(element.text).to eq 'New Digital Object Digital Object'
+
+    fill_in 'digital_object_title_', with: "Digital Object Title #{now}"
+    fill_in 'digital_object_digital_object_id_', with: "Digital Object Identifier #{now}"
+    select 'Mixed Materials', from: 'digital_object_digital_object_type_'
+
+    click_on 'Add File Version'
+    fill_in 'digital_object_file_versions__0__file_uri_', with: "File Version URI #{now}"
+    element = find('#digital_object_file_versions__0__file_size_bytes_')
+    element.fill_in with: '100'
+
+    # Click on save
+    find('button', text: 'Save Digital Object', match: :first).click
+
+    element = find('.alert.alert-success.with-hide-alert')
+    expect(element.text).to eq "Digital Object Digital Object Title #{now} Created"
+    element = find('.table-row.root-row .title')
+    expect(element.text).to eq "Digital Object Title #{now}"
+  end
+
+  it 'can handle multiple file versions and file system and network path types' do
+    now = Time.now.to_i
+
+    click_on 'Create'
+    click_on 'Digital Object'
+    element = find('h2')
+    expect(element.text).to eq 'New Digital Object Digital Object'
+
+    fill_in 'digital_object_title_', with: "Digital Object Title #{now}"
+    fill_in 'digital_object_digital_object_id_', with: "Digital Object Identifier #{now}"
+    select 'Mixed Materials', from: 'digital_object_digital_object_type_'
+
+    click_on 'Add File Version'
+    fill_in 'digital_object_file_versions__0__file_uri_', with: "File Version URI 1 #{now}"
+    element = find('#digital_object_file_versions__0__file_size_bytes_')
+    element.fill_in with: '100'
+
+    # Click on save
+    element = find('button', text: 'Save Digital Object', match: :first).click
+    element = find('.alert.alert-success.with-hide-alert')
+    expect(element.text).to eq "Digital Object Digital Object Title #{now} Created"
+
+    click_on 'Add File Version'
+    fill_in 'digital_object_file_versions__1__file_uri_', with: "File Version URI 2 #{now}"
+    element = find('#digital_object_file_versions__1__file_size_bytes_')
+    element.fill_in with: '200'
+
+    # Click on save
+    element = find('button', text: 'Save Digital Object', match: :first).click
+
+    while true do
+      sleep 1
+      break if page.evaluate_script('jQuery.active') == 0
+    end
+
+    element = find('.alert.alert-success.with-hide-alert')
+    expect(element.text).to eq "Digital Object Digital Object Title #{now} Updated"
+
+    click_on 'Close Record'
+
+    element = find('h2')
+    expect(element.text).to eq "Digital Object Title #{now} Digital Object"
+    expect(page).to have_css 'h3', text: 'File Versions'
+    elements = all('#digital_object_file_versions__accordion .panel.panel-default')
+    expect(elements.length).to eq 2
+
+    expect(elements[0]).to have_text "File Version URI 1 #{now}"
+    expect(elements[1]).to have_text "File Version URI 2 #{now}"
+  end
```

Comment:
The file version URIs as defined above on lines 60 and 70 don't test for the 'and file system and network path types' requirement of this spec. See the original spec that tried to account for windows, unix, and local weird file path.